.circleci/config.yml

```diff
@@ -94,7 +94,10 @@ jobs:
             sleep 20s && \
             cd end-to-end-tests && \
             npm install && \
-            npm run test-travis
+            npm install -g webdriver-manager && \
+            webdriver-manager update && \
+            webdriver-manager start && \
```

Comment:
you'll want to start that one in the background, run with `&`. Then prolly sleep 1s or so after